codecov.yml

```diff
@@ -0,0 +1,24 @@
+# @file codecov.yml
+#
+# Codecov configuration file
+#
+##
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: Apache-2.0
+##
+codecov:
+  require_ci_to_pass: true
+coverage:
+  precision: 2
+  round: nearest
+  range: 80..90 # 0-79% red, 80-89% yellow, 90-100% green
+  status:
+    project:
+      default:
+        target: 80% # Target 80% coverage for the project

```

Comment:
Considering the amount of uncoverable code we have, Is the assumption here that devs should tag such code with `#coverage(off)`?

Reply:
> Considering the amount of uncoverable code we have, Is the assumption here that devs should tag such code with `#coverage(off)`?

I mean that is exactly why the coverage target is 80% instead of 100%, however yes, you can also use coverage(off). I'll update the docs and have you re-review.

Reply:
@makubacki @cfernald I've updated docs.